<launch>

  <!-- <node name="joy_node" pkg="joy" type="joy_node" >
  <param name="autorepeat_rate" type="double" value="100.0" />
  </node> -->

  <node name="emg2servo" pkg="direct_control" type="emg2servo" output="screen" >
    <!-- <remap from="/joint_states" to="/mpl_right_arm/wrapped_joint_states" /> -->
    <remap from="/emg2servo/mpl_right_arm/arm_controller/command" to="/mpl_right_arm/arm_controller/command" />
    <!-- <remap from="/emg2servo/servo_server/delta_joint_cmds" to="/servo_server/delta_joint_cmds" />  -->
  </node>

</launch>